openshift/Dockerfile.requirements

```diff
@@ -1,4 +1,4 @@
-FROM registry.ci.openshift.org/ocp/4.17:base-rhel9
+FROM registry.ci.openshift.org/ocp/4.16:base-rhel9
```

Comment:
Sorry to ask now, but the reason behind not having this Dockerfile inside the release config is that this image is supported to be built and run locally only, right?

Reply:
Yes, that's correct.

Reply:
But, I guess we can get it added to the CI config so that ART bot creates the bump PR for this Dockerfile as well.

Reply:
Yeah, having some auto reminders is always helpful. We can do it as part of a separate chore.